### PR TITLE
Added Myntora

### PR DESCRIPTION
**Pull request by RyanMoodGAMING**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [x] `primaryAddress`: an address *(ex. `my.server`)*
  - [x] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]`)*
  - [x] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18`)*

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image's file name matches its mapping id.
* [x] My image is `512` pixels in width and height.

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -1795,4 +1795,14 @@
         "minecraftVersions": ["1.7", "1.8"],
         "primaryMinecraftVersion": "1.8"
     }
+    {
+        "id": "myntora",
+        "name": "Myntora",
+        "addresses": [
+            "play.myntora.net"
```
Comment (itsJordan) [inline servers.json:1802]:
```suggestion
            "myntora.net"
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please add your server logo to the 'logos' directory

**Comment by RyanMoodGAMING** [comment]
Will do once I’ve been sent to them.

**Comment by itsJordan** [comment]
> Will do once I’ve been sent to them.

Do you have any idea when you will have access to the logo?

**Comment by RyanMoodGAMING** [comment]
Uploaded.

File: servers.json [hunk 2]
```diff
@@ -1795,4 +1795,14 @@
         "minecraftVersions": ["1.7", "1.8"],
         "primaryMinecraftVersion": "1.8"
     }
```
Comment (itsJordan) [inline servers.json:1797]:
```suggestion
    },
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please review my suggestion to validate your json.

**Comment by RyanMoodGAMING** [comment]
Suggestion added.

File: servers.json [hunk 3]
```diff
@@ -1794,5 +1794,15 @@
...
         "primaryMinecraftVersion": "1.8"
+    },
+    {
+        "id": "myntora",
+        "name": "Myntora",
+        "addresses": [
+            "myntora.net"
+        ],
+        "primaryAddress": "play.myntora.net",
+        "minecraftVersions": ["1.8", "1.9", "1.12", "1.16", "1.18", "1.19"],
```
Comment (itsJordan) [inline servers.json:1805]:
```suggestion
        "minecraftVersions": ["1.8", "1.12", "1.16", "1.18", "1.19"],
```

**Review by itsJordan — CHANGES_REQUESTED** [review]
1.9 is not supported on Lunar.

**Comment by RyanMoodGAMING** [comment]
My bad, I thought it meant versions supported in general not lunar versions.

**Review by itsJordan — APPROVED** [review]

**Pull request merged**